cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py

```diff
@@ -1,33 +1,42 @@
 import unittest
 from unittest import mock
 from pathlib import Path
-from tempfile import NamedTemporaryFile
+from tempfile import TemporaryDirectory
+from contextlib import contextmanager
 
 from cumulusci.core.exceptions import TaskOptionsError
 from cumulusci.tasks.bulkdata.tests.utils import _make_task
 
 import yaml
 from sqlalchemy import create_engine
 
-try:
-    import snowfakery
-    from cumulusci.tasks.bulkdata.generate_and_load_data_from_yaml import (
-        GenerateAndLoadDataFromYaml,
-    )
-except ImportError:
-    snowfakery = None
+import snowfakery
+from cumulusci.tasks.bulkdata.generate_and_load_data_from_yaml import (
+    GenerateAndLoadDataFromYaml,
+)
+from snowfakery import data_generator_runtime
 
 if snowfakery:
```

Comment:
This condition can go too

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1659/commits/2814208b6099beb23b5812d6d36207792fdff931